features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumResourceScope.kt

```diff
@@ -214,11 +216,48 @@ internal class RumResourceScope(
         val finalTiming = timing ?: extractResourceTiming(
             attributes.remove(RumAttributes.RESOURCE_TIMINGS) as? Map<String, Any?>
         )
+
+        val rumFeatureContext = sdkCore.getFeatureContext(RUM_FEATURE_NAME)
```

Comment:
if context is needed, we should get it from `newRumEventWriteOperation(writer) { datadogContext` below